src/pages/community-champions.js

```diff
@@ -470,6 +490,137 @@ const OtherDetailsAndRequirements = () => (
+    padding: ${size.xlarge} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        margin: ${size.large} -${size.default};
+        padding: ${size.large} ${size.default};
+    }
+`;
+
+const ChampionsContainer = styled('div')`
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    row-gap: ${size.default};
+    @media ${screenSize.upToLarge} {
+        row-gap: ${size.xsmall};
+    } ;
+`;
+
+const ChampionItemContainer = styled('div')`
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    padding: ${size.default};
+    text-align: center;
+    width: ${CHAMPION_ITEM_CONTAINER_WIDTH};
+    @media ${screenSize.upToMedium} {
+        padding: ${size.default} 0;
+        width: ${CHAMPION_ITEM_CONTAINER_MOBILE_WIDTH};
+    }
+`;
+
+const LocationContainer = styled('div')`
+    display: flex;
+    justify-content: center;
+    margin-top: ${size.tiny};
+`;
+
+const PinLocationImage = styled('img')`
+    margin-right: ${size.tiny};
+`;
+
+const LocationText = styled(P4)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+`;
+
+const ChampionLocation = ({ location }) => (
+    <LocationContainer>
+        <PinLocationImage src={PinLocationIcon} />
+        <LocationText collapse>{location}</LocationText>
+    </LocationContainer>
+);
+
+const ChampionTitleText = styled(P3)`
+    color: ${({ theme }) => theme.colorMap.greyLightTwo};
+`;
+
+const ChampionProfilePicture = styled(ProfileImage)`
+    margin-bottom: ${size.default};
+    div {
+        background-size: cover;
+    }
+`;
+
+const ChampionItem = ({ imageUrl, location, name, title }) => (
+    <ChampionItemContainer>
+        <ChampionProfilePicture
+            defaultImage={ChampionPlaceholderImage}
+            gradientOffset={CHAMPION_PROFILE_PICTURE_GRADIENT_OFFSET}
+            hideOnMobile={false}
+            image={imageUrl}
+            key={name}
+            height={CHAMPION_PROFILE_PICTURE_SIZE}
+            width={CHAMPION_PROFILE_PICTURE_SIZE}
+        />
+        <H6 collapse>{name}</H6>
+        <ChampionTitleText collapse>{title}</ChampionTitleText>
+        <ChampionLocation location={location} />
+    </ChampionItemContainer>
+);
+
+const communityChampions = graphql`
+    query CommunityChampions {
+        allStrapiCommunityChampions {
+            nodes {
+                id
+                firstName
+                middleName
+                lastName
+                location
+                title
+                image {
+                    url
+                }
+            }
+        }
+    }
+`;
+
+const ChampionList = () => {
+    const data = useStaticQuery(communityChampions);
+    const champions = dlv(
+        data,
+        ['allStrapiCommunityChampions', 'nodes'],
+        []
+    ).map(champion => {
+        champion.fullName = [
+            champion.firstName,
+            champion.middleName,
+            champion.lastName,
+        ].join(' ');
+        return champion;
```

Comment:
```suggestion
        return {...champion, fullName: `${champion.firstName} ${champion.middleName} ${champion.lastName}`}
```

^ although this would give an extra space if no middle name is given here is a shorthand that also would help!

Reply:
doing the sort as part of my graphql query eliminates the need for this fullName field so i've removed this